docs/current_docs/api/module-structure.mdx

```diff
@@ -165,12 +168,35 @@ py-modules = ["main"]
 </TabItem>
 </Tabs>
 
-:::note
-Build backends are independent from installers (or *build frontends*, see [PEP 517](https://www.python.org/dev/peps/pep-0517/)), even though some installers may provide both (as separate packages). For example, [hatch](https://pypi.org/project/hatch/) vs [hatchling](https://pypi.org/project/hatchling/), [poetry](https://pypi.org/project/poetry/) vs [poetry-core](https://pypi.org/project/poetry-core/).
 :::
 
-:::tip
-`dagger init` won't override existing `pyproject.toml` and `.py` files, so it's possible to use an external process to generate a different template, before calling `dagger init`.
+:::tip CUSTOMIZE IMPORT PACKAGE
+The Python SDK looks for the main object of the Dagger module in the Pyhton [import package that is named after the distribution package name](https://packaging.python.org/en/latest/discussions/distribution-package-vs-import-package/#how-do-distribution-package-names-and-import-package-names-compare) (in particular, using underscores `_` as a word separator).
+
+If they are different, you explicitly tell the Python SDK where the main object needs to be imported from, using the following entry point configuration in `pyproject.toml`:
+
+```toml
+[project.entry-points."dagger.mod"]
+main_object = "<import package>:<main object>"
+````
+
+For example, for a Dagger module named `my-module`:
```

Comment:
I'm not sure why but just noting that when viewing this PR in the GitHub UI, this and following lines are highlighted in red. I'm not sure if that indicates an error in formatting or some GitHub diff weirdness.

Reply:
I think that's because of the extra backtick just above.

Reply:
Yeah, it's an issue with the backticks.